src/wireviz/wv_colors.py

```diff
@@ -139,3 +139,12 @@ def translate_color(input, color_mode):
         return output.upper()
     else:
         return output.lower()
+
+
+def html_color_icon(inp):
+    if inp == None:  # no color specified
+        return ''
+    elif inp == "WH":  # color matches background color
+        return '&#x25A1;'  # square outline in default text color
```

Comment:
It would be lovely to have black borders around all colors.

I looked at some unicode hackery, using stuff like the `&#x20DD;` combining enclosing circle (test [here](https://www.w3schools.com/code/tryit.asp?filename=GHA5WF2CUFP4)) but it's quite there yet.
I did find this [StackOverflow thread](https://stackoverflow.com/questions/57528146) with a hilarious twist at the end, though... 😁 

I'm very open to other suggestions! I must say, though, that I really like the look of the small color dots, so it would be nice to somehow preserve that... but feel free to suggest other things as well.

Reply:
How about using the `connector.color` as background color in the icon column? That probably should reflect the real color contrasts of the physical connector. In that case you also will need something like this:
```py
def html_color_icon(code, bgcode = "WH"):
    if code in [None, '__']:  # no color specified
        return ''
    elif code == bgcode:  # color matches background color
        return '&#x25CB;'  # circle outline in default text color
    else:
        return f'<font color="{translate_color(code, "HEX")}">&#x25CF;</font>'  # filled circle in requested color
```
![pr141](https://user-images.githubusercontent.com/12657033/89029068-de9c5d80-d32d-11ea-8f82-038055fc1ce4.png)

Reply:
interesting suggestion, but such big color blocks might be a bit visually overwhelming, especially if you have connectors with very dark colors... and there is always going to be some color combinations with poor contrast, I guess.

Reply:
> interesting suggestion, but such big color blocks might be a bit visually overwhelming, especially if you have connectors with very dark colors... and there is always going to be some color combinations with poor contrast, I guess.

I see your points, and I don't disagree with you, but I had to share this idea anyway -  it might trigger some other ideas later. 😃